Utilities/Thread.cpp

```diff
@@ -2868,6 +2872,65 @@ void thread_ctrl::detect_cpu_layout()
 	}
 }
 
+#if defined(ARCH_ARM64) && !defined(__APPLE__)
+
+void detect_big_little_topology(std::vector<int>& big_cores, std::vector<int>& little_cores, std::vector<int>& prime_cores)
+{
+    big_cores.clear();
+    little_cores.clear();
+    prime_cores.clear();
+
+    const std::string base_path = "/sys/devices/system/cpu/";
+
+    for (unsigned int i = 0; i < 8; ++i)
```

Comment:
Rpcs3 works (and is being tested by users) on asahi linux as well as ampere boards at the moment which have many more cores than this.

Reply:
On arm CPU?